cpt-squid.rst

```diff
@@ -60,11 +60,19 @@ something like this:
 
       acl localnet src A.B.C.D/NN
 
-If you instead want to limit the destinations to major cvmfs Stratum 1s,
-it is better to use the list built in to 
+If you instead want to limit the destinations to major WLCG cvmfs Stratum 1s,
+it is best to use the list built in to
 `Frontier Squid <https://twiki.cern.ch/twiki/bin/view/Frontier/InstallSquid#Restricting_the_destination>`_
 because the list is sometimes updated with new releases.
 
+For manually limiting the destinations, use a configuration like
+
+::
+
+      acl cvmfs dst <a stratum 1 host name>
+      acl cvmfs dst <another stratum 1 host name>
```

Comment:
I recommend instead using dstdom_regex because it does a string match instead of looking up the names in the DNS and caching them.  That makes it able to tolerate chages in IP addresses of the destination name without having to reload squid.  So the example would be:
```
acl cvmfshost dstdom_regex ^(a\.stratum1\.host|another\.stratum1\.host)$
```
I'm not sure if there can be mutiple regexes for the same name, I haven't tried it.  Probably there could be.

Reply:
Or maybe a better example would be since those two are in the same "domain":
```
acl cvmfshost dstdom_regex ^(a|another)\.stratum1\.host)$
```